exe/kubernetes-deploy

```diff
@@ -13,10 +13,9 @@ verbose_log_prefix = false
 max_watch_seconds = nil
 
 ARGV.options do |opts|
+  binds = []
   opts.on("--bindings=BINDINGS", "Expose additional variables to ERB templates " \
-    "(format: k1=v1,k2=v2, JSON string or file (JSON or YAML) path prefixed by '@')") do |binds|
-    bindings.merge!(KubernetesDeploy::BindingsParser.parse(binds))
```

Comment:
any reason not to use `deep_merge!` here?

Reply:
That was my original change but after realising its done in two files and testing would be super funky it seemed to make sense extract. I'm fine either way but not sure how clean we can make tests if its just changed in the exes